docs/source/customizations/coprocessor.mdx

```diff
@@ -580,48 +578,65 @@ A unique ID corresponding to the client request associated with this coprocessor
 <tr>
 <td>
 
-##### `serviceName`
+##### `body`
 
-`string`
+`string | object`
 
 </td>
 <td>
 
-The name of the subgraph that this coprocessor request pertains to.
+The body of the corresponding request or response.
 
-This value is present only for coprocessor requests from the router's `SubgraphService`.
+If your coprocessor [returns a _different_ value](#responding-to-coprocessor-requests) for `body`, the router replaces the existing body with that value. This is common when [terminating a client request](#terminating-a-client-request).
 
-**Do not return a _different_ value for this property.** If you do, the router treats the coprocessor request as if it failed.
+This field's type depends on the coprocessor request's [`stage`](#stage):
+
+- For `SubgraphService` stages, `body` is a JSON _object_.
+- For `RouterService` stages, `body` is a _string_.
+  - This is necessary to support handling [deferred queries](#handling-deferred-query-responses).
+  - If you modify `body` during the `RouterRequest` stage, the new value must be a valid string serialization of a JSON object. If it isn't, the router detects that the body is malformed and returns an error to the client.
+
+This field's structure depends on whether the coprocessor request corresponds to a request, a standard response, or a response "chunk" for a deferred query:
+
+- **If a request,** `body` usually contains a `query` property containing the GraphQL query string.
+- **If a standard response,** `body` usually contains `data` and/or `errors` properties for the GraphQL operation result.
+- **If a response "chunk",** `body` contains `data` for _some_ of the operation fields.
```

Comment:
I want to provide more context about how this body differs from a standard response body, but it got late

Reply:
Ok. I struggled with how best to describe this myself. We can work on that.